src/indexing.jl

```diff
@@ -54,7 +54,7 @@ end
 function _getindex(t::IndexedTable, idxs)
     I = t.index
     cs = astuple(I.columns)
-    if length(idxs) != length(I.columns)
+    if nfields(idxs) !== nfields(I.columns)
```

Comment:
Doesn't `length` work on both? Better to use that since it's a more general function.